internal/server/v1/propertyvalues/golden/bulk_linked_property_values/result.json

```diff
@@ -10503,7 +10503,7 @@
           "dcid": "geoId/0640312"
         },
         {
-          "name": "La Presa, California",
+          "name": "La Presa",
```

Comment:
Due to the new ordering, the shorter name is used now:

![image](https://user-images.githubusercontent.com/5951856/184955400-3926a57a-c978-4e9e-bb70-1f36eb6ba592.png)

Reply:
Interesting.

Reply:
Hmmm, though name string shouldn't be part of the key?  And this PR changes only the key?

Reply:
The key is "dcid" + "!" + "value". When it's a leaf value, "dcid" is empty so ordering is based on value only.

Reply:
Ah, I see.

`^La Presa, California^dc/prov` < `^La Presa^dc/prov`

`^La Presa!dc/prov` < `^La Presa, California!dc/prov`